cicecore/cicedyn/infrastructure/io/io_pio2/ice_restart.F90

```diff
@@ -57,13 +59,19 @@ subroutine init_restart_read(ice_ic)
       integer (kind=int_kind) :: status
 
       logical (kind=log_kind), save :: first_call = .true.
-
+      logical :: exist
       character(len=*), parameter :: subname = '(init_restart_read)'
 
       if (present(ice_ic)) then
          filename = trim(ice_ic)
       else
          if (my_task == master_task) then
+#ifdef CESMCOUPLED
+            write(pointer_file,'(a,i4.4,a,i2.2,a,i2.2,a,i5.5)') &
+                 'rpointer.ice'//trim(inst_suffix)//'.',myear,'-',mmonth,'-',mday,'-',msec
```

Comment:
Does this duplicate https://github.com/ESCOMP/CICE/blob/6d8c7f68c2ef80da9973f27c0077e54018bdbbfe/cicecore/cicedyn/general/ice_init.F90#L1202
 ?
which says:
```
#ifdef CESMCOUPLED
      pointer_file = trim(pointer_file) // trim(inst_suffix)
#endif

Reply:
No.   That adds the inst_suffix which allows for single executable ensembles.  This adds a date stamp.

Reply:
In ice_init it uses `pointer_file` from the ice_in namelist but here it's hardcoded to 'rpointer.ice', I dont think we need both

Reply:
I think either: 
- in ice_init you can abort with "'pointer_file' namelist not used in CESMCOUPLED, 
- use 'pointer_file' in ice_restart.F90 and remove the three lines in ice_init

Reply:
So, our thought here is to add lpointer_file for the purposes of the multiple instances. When not in CESM:

lpointer_file = pointer_file

Does this work?

Reply:
Why not just this ?:

The inst_suffix was already added in ice_init 

```
#ifdef CESMCOUPLED
            write(pointer_file,'(a,i4.4,a,i2.2,a,i2.2,a,i5.5)') &
                 pointer_file//'.',myear,'-',mmonth,'-',mday,'-',msec
            inquire(file=pointer_file, exist=file_exist)
            if (.not. file_exist) pointer_file = 'rpointer.ice'//trim(inst_suffix)
#endif
```

Reply:
Because that ends up repeating the timestamp if you are starting from a restart.

Reply:
Oh got it - thanks both, adding lpointer_file sounds good.